src/helpers/validateEmailAddress.js

```diff
@@ -1,4 +1,17 @@
-//TODO
-export default () => {
-  return true;
+const ALPHANUMERIC =
+  "a-zA-Z0-9àèìòùÀÈÌÒÙáéíóúýÁÉÍÓÚÝâêîôûÂÊÎÔÛãñõÃÑÕäëïöüÿÄËÏÖÜŸçÇßØøÅåÆæœ" + // Number and letters including accented characters
+  "\u3000-\u303F\u3040-\u309F\u30A0-\u30FF\uFF00-\uFFEF\u4E00-\u9FAF\u2605-\u2606\u2190-\u2195\u203B"; // Chinese/Japanese/Korean characters
+
+const EMAIL_VALIDATION_STRING =
+  `^[${ALPHANUMERIC}.!#$%&'*+/=?^_\`{|}~-]` + // Local part
+  `+@[${ALPHANUMERIC}](?:[${ALPHANUMERIC}-]` + // Domain name
+  `{0,61}[${ALPHANUMERIC}])?(?:\\.` + // Allow periods in domain name
+  `[${ALPHANUMERIC}](?:[${ALPHANUMERIC}-]{0,61}[${ALPHANUMERIC}])?)*\\.` +
+  `[${ALPHANUMERIC}](?:[${ALPHANUMERIC}-]{0,61}` + // End of domain
+  `[${ALPHANUMERIC}])$`;
```

Comment:
This is pretty gross.. Is there an email address validation library we can use instead? :/

Reply:
I've just seen you have installed one in the package.json - did it not work out?

Reply:
Discussed on Slack 💬

Reply:
For propsperity

> 
> Steve Thomson  14 minutes ago
> The reason we have done it manually is that there were problems with GovNotify validation not being consistent with validation libraries on the DFE teacher training project. This would cause bad user experience if the form said valid and then we got a rejection from GovNotify. As we already had an implementation we could steal we went for it. https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/960/files
> :this:
> 1
> 
> 
> Antony O'Neill  11 minutes ago
> :thinking_face:  Do we use error messages from GovNotify to determine validity of phone numbers?
> 
> Steve Thomson  10 minutes ago
> Yes and no. We validate using a library but then GN  will throw an error later if it doesn't like the number. You can see this happening if you try a non-team phone number on staging. (edited) 
> 
> Steve Thomson  10 minutes ago
> Which would return a 400 from the api to the client, which we then don't handle super gracefully imo. I think I made a ticket to look at it.
> :+1:
> 1
> 
>